src/main/rx/ghst.c

```diff
@@ -320,32 +311,36 @@ static void ghstIdle()
 
 bool ghstRxInit(const rxConfig_t *rxConfig, rxRuntimeState_t *rxRuntimeState)
 {
-    for (int iChan = 0; iChan < GHST_MAX_NUM_CHANNELS; ++iChan) {
-        ghstChannelData[iChan] = (16 * rxConfig->midrc) / 10 - 1408;
-    }
-
     rxRuntimeState->channelCount = GHST_MAX_NUM_CHANNELS;
     rxRuntimeState->rxRefreshRate = GHST_TIME_BETWEEN_FRAMES_US;
 
     rxRuntimeState->rcReadRawFn = ghstReadRawRC;
     rxRuntimeState->rcFrameStatusFn = ghstFrameStatus;
-    rxRuntimeState->rcFrameTimeUsFn = ghstFrameTimeUs;
+    rxRuntimeState->rcFrameTimeUsFn = rxFrameTimeUs;
     rxRuntimeState->rcProcessFrameFn = ghstProcessFrame;
 
+    for (int iChan = 0; iChan < rxRuntimeState->channelCount; ++iChan) {
+        if( iChan < 4 ) {
```

Comment:
Have seen this before in the other PR:
```suggestion
        if (iChan < 4 ) {
```